pegasus/sites.v3/code.org/public/css/homepage.css

```diff
@@ -355,6 +355,25 @@ body.modal-open .supreme-container{
   margin: 10px 5px 0 0;
 }
 
+#homepage #actions .herolink-white {
+  font-family: 'Gotham 4r', sans-serif;
+  font-weight: 500;
+  font-size: 18px;
+  color: white;
+  display: inline-block;
+  margin-top: 24px
+}
+
+#homepage #actions .herolink-white i {
+  vertical-align: middle;
+  margin-right: 5px;
+  font-size: 26px;
```

Comment:
Just curious, was 26px just the size that made things look right or was there a reason for picking this value?

Reply:
This is specifically for the icon style `i`, I just looked at the mocks in the ticket and it was the size that looked most like the icon in the mocks.